Makefile

```diff
@@ -0,0 +1,9 @@
+venv: requirements-dev.txt requirements-test.txt
+	python3 -m venv venv
+
+	. venv/bin/activate; \
+	pip3 install -r requirements-dev.txt; \
```

Comment:
how does the package itself get installed in this `venv`? should there be a `pip install -r requirements.txt` too?

Reply:
perf